src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalController.java

```diff
@@ -638,4 +639,39 @@ public ResponseEntity<Object> updateAnOrganisationsRegisteredPbas(
                 .status(updatePbaStatusResponse.getStatusCode())
                 .body(updatePbaStatusResponse);
     }
+
+    @Operation(
+            summary = "Retrieves the organisation details of a user",
+            description = "**IDAM Roles to access API** : <br> prd-admin",
+            security = {
+                    @SecurityRequirement(name = "ServiceAuthorization"),
+                    @SecurityRequirement(name = "Authorization")
+            }
+    )
+
+    @ApiResponse(
+            responseCode = "200",
+            description = "",
+            content = @Content(schema = @Schema(implementation = OrganisationEntityResponse.class))
+    )
+    @ApiResponse(
+            responseCode = "403",
+            description = "Forbidden Error: Access denied",
+            content = @Content
+    )
+    @ApiResponse(
+            responseCode = "500",
+            description = "Internal Server Error",
+            content = @Content
+    )
+
+    @GetMapping(
+            path = "/orgDetails/{userId}",
+            produces = APPLICATION_JSON_VALUE
+    )
+    @Secured("prd-admin")
+    public ResponseEntity<OrganisationEntityResponse> retrieveOrganisationByUserId(
+            @PathVariable("userId") @Valid @NotNull @NotBlank String userId) {

```

Comment:
annotations @Valid @NotNull @NotBlank are not effective , pls remove them. as service method validating.

Reply:
done